src/bentoml/_internal/service/inference_api.py

```diff
@@ -126,6 +127,17 @@ def __init__(
                                 f"Expected type of argument '{second_arg}' to be '{input_type}', got '{sig.parameters[second_arg].annotation}'"
                             )
 
+        if user_defined_callback is not None:
+            self.func = user_defined_callback
+        else:
+
+            def nop(*args: t.Any, **kwargs: t.Any):
+                return
+
+            self.func = nop
```

Comment:
Was there a problem with just
```suggestion
        self.func = lambda _: None
```

Reply:
nothing. Just preference.